framework/src/modules/chain/chain.js

```diff
@@ -235,7 +235,7 @@ module.exports = class Chain {
 			if (!this.options.loading.rebuildUpToRound) {
 				this.channel.subscribe(
 					'network:event',
-					async ({ data: { event, data, peerId } }) => {
+					async ({ packet: { event, data, peerId } }) => {
```

Comment:
you cannot do this because `data` is wrapped object from the channel.
In this case, root level `data` will not be variable you can use, so it won't be shadowed too